### PR TITLE
feat(github) : Add github files

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
- Add github files templates in order to follow the org standard in order to follow org standard and the community practices.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<!--- Insert the output of `go version` here -->
+
+* Kubernetes version information:
+
+<!--- Insert the output of `kubectl version` here -->
+
+* Kubernetes cluster kind: 
+
+* Are you writing your operator in ansible, helm, or go?
```
Comment (ecordell) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Not sure this is relevant to all OLM users?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+**What did you see instead? Under which circumstances?**
+A clear and concise description of what you expected to happen (or insert a code snippet).
+
+
+**Environment**
+* operator-lifecycle-manager version:
+
+<!--- Insert operator-lifecycle-manager release or Git SHA here. -->
+
+* go version:
```
Comment (ecordell) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Not sure this is relevant to OLM users

Comment (camilamacedo86) [reply .github/ISSUE_TEMPLATE/bug-report.md]: 👍

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (camilamacedo86) [reply .github/ISSUE_TEMPLATE/bug-report.md]: 👍

**Comment by camilamacedo86** [comment]
Hi @ecordell, 

Really thank you for your time and review. the changes requested are done with success.

**Comment by camilamacedo86** [comment]
Hi @ecordell,

It is rebased with the master, could we merge it now?

**Comment by ecordell** [comment]
doc only - merge

**Pull request merged**